src/app/right-hand-panel.js

```diff
@@ -13,20 +31,47 @@ import 'bootstrap';
 @customElement('right-hand-panel')
 @inject(Context, Element)
 export class RightHandPanel {
+
+    tabs = [
```

Comment:
btw. I found some good resource online which can maybe help to explain a bit more what I was trying to say: https://www.danyow.net/aurelia-custom-element-vs-compose/